GVFS/GVFS.Service/GVFSMountProcess.Mac.cs

```diff
@@ -1,52 +1,63 @@
 ﻿using GVFS.Common;
-using GVFS.Common.Tracing;
-using GVFS.Service.Handlers;
-using System;
-
+using GVFS.Common.Tracing;
+using System.Diagnostics;
+
 namespace GVFS.Service
 {
-    public class GVFSMountProcess : IDisposable
+    public class GVFSMountProcess : IRepoMounter

```

Comment:
I suggested the interface name because the interface shouldn't be a description of the class(es) that implement the interface and the only method that is in the interface is `MountRepository` so if we wanted to pull out this code or have another class be able to mount repositories it wouldn't have anything to do with a Process.